lib/rules/typescript-uses-vars.js

```diff
@@ -0,0 +1,154 @@
+/**
```

Comment:
I don't think we need the word "typescript" in the rule, since the plugin is already called "typescript". Maybe just `no-unused-vars` to match the name of the core rule?